README.md

```diff
@@ -7,7 +7,9 @@ It provides real, authenticated Windows Auth capabilities - not just a mock of s
 
 ## Getting Started
 
-There is currently no NuGet package for this project. Open an issue if this project is of interest to you and we'll look at publishing a NuGet package. Otherwise, you can add this project as a git submodule to try it out.
+[![Build Status](https://intellitect.visualstudio.com/IntelliTect/_apis/build/status/IntelliTect.AspNetCore.TestHost.WindowsAuth-Nuget-Publish?branchName=master)](https://intellitect.visualstudio.com/IntelliTect/_build/latest?definitionId=50?branchName=master)
```

Comment:
https://intellitect.visualstudio.com/IntelliTect/_build/latest?definitionId=50?branchName=master is requiring a sign-in - looks like https://intellitect.visualstudio.com/IntelliTect is private, so this link isn't viewable by anyone who isn't an IntelliTect person. Interesting, the build badge is still viewable without a sign-in.

We may want to make a new project on https://intellitect.visualstudio.com for builds of our miscellaneous GitHub projects - something like "IntelliTectPublic" perhaps.

Reply:
Are public projects a generally-available feature in DevOps?

Reply:
Yes. Coalesce is, for example: https://intellitect.visualstudio.com/coalesce (open in incognito).